### PR TITLE
Bidtracker Status tooltips TM-2020

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
~Currently just rendering for draft.~

~Not sure why React-Tippy Light Theme is a bright blue.~ overriding in src/sass/_tooltip.scss; i think we may have been overriding bc react-tippy className prop doesn't seem to work/apply classes to the container. Woot discussed solution with Mike!

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
src/Components/PositionTitle/PositionTitle.jsx uses react-tippy theme light, with the blue.

File: src/Components/BidTracker/BidPhaseConfig/Complete.js [hunk 1]
```diff
@@ -64,25 +65,38 @@ export default function bidClassesFromCurrentStatus(bid = { status: 'draft' }) {
...
       bidClassObject.stages[SUBMITTED_PROP] = {
         ...DEFAULT_INCOMPLETE_OBJECT,
         date: SUBMITTED_DATE,
         needsAction: true,
         isCurrent: true,
         isPendingLine: true,
         title: SUBMIT_BID_ACTION_TITLE,
-        number: SUBMITTED_NUMBER };
+        number: SUBMITTED_NUMBER,
+        tooltip: SUBMITTED_TOOLTIP };
```
Comment (mjoyce91) [inline src/Components/BidTracker/BidPhaseConfig/Complete.js:93]: I'm not sure this is what you want if you're going to be use react-tippy

Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidPhaseConfig/Complete.js:93]: Oh? How come?

File: src/Components/BidTracker/BidStep/BidStepIcon/BidStepIcon.jsx [hunk 2]
```diff
@@ -20,34 +22,87 @@ const assignClasses = (isComplete, needsAction, isCurrent) => {
...
-        >
-          {number > 0 ? number : null}
-        </span>
-        { hasRescheduledTooltip && <RescheduledIcon />}
-      </div> :
-      <FontAwesome name="check" /> }
+        ?
+        <div className="icon-container">
+          { tooltipTitle && tooltipText && !condensedView ?
+              <Tooltip
```
Comment (mjoyce91) [inline src/Components/BidTracker/BidStep/BidStepIcon/BidStepIcon.jsx:59]: Can we have the tooltip always appear above the text?

File: src/Constants/BidData.js [hunk 3]
```diff
@@ -72,3 +72,30 @@ export const BID_TRACKER_SUBMITTED_ACTIVE_STATUSES = [
   APPROVED_PROP, HAND_SHAKE_ACCEPTED_PROP, HAND_SHAKE_OFFERED_PROP,
   PRE_PANEL_PROP, IN_PANEL_PROP, SUBMITTED_PROP, PANEL_RESCHEDULED_PROP,
   HAND_SHAKE_NEEDS_REGISTER_PROP];
+
+export const BID_TRACKER_TOOLTIP_TEXT = {
+  [DRAFT_PROP]: {
+    title: `What is a "${GET_DRAFT_TITLE()}"?`,
+    text: `"${GET_DRAFT_TITLE()}" are bids you have on your bid tracker but you haven’t submitted them for consideration.`,
```
Comment (mjoyce91) [inline src/Constants/BidData.js:79]: Wording seems better as plural:

```jsx
"${GET_DRAFT_TITLE()}s"
```

File: src/Constants/BidData.js [hunk 4]
```diff
@@ -72,3 +72,30 @@ export const BID_TRACKER_SUBMITTED_ACTIVE_STATUSES = [
...
+    title: `What is "${GET_HAND_SHAKE_COMPLETE_REGISTER_TITLE()}"?`,
+    text: `"${GET_HAND_SHAKE_COMPLETE_REGISTER_TITLE()}" reflects the date your handshake was registered by the CDO. Next the GTM/CDA Assignment Officer for your onward post prepares your assignment for panel.`,
+  },
+  [PRE_PANEL_PROP]: {
+    title: `What is "${GET_PANEL_TITLE()}"?`,
+    text: `"${GET_PANEL_TITLE()}" displays the date you are scheduled to be paneled once your position has been added to the panel agenda.`,
+  },
+  [APPROVED_PROP]: {
+    title: `What is "${GET_APPROVAL_TITLE()}"?`,
+    text: `"${GET_APPROVAL_TITLE()}" is the date your assignment was officially approved at panel (Hover over this status for a surprise!) Within a few days of being approved at panel, you will receive your Travel Message (TMOne) announcing to your current and future post your approved assignment.`,
```
Comment (mjoyce91) [inline src/Constants/BidData.js:99]: Needs a period after the parens

**Comment by mjoyce91** [comment]
Looks good, just a few comments

File: src/Components/BidTracker/BidStep/BidStepIcon/BidStepIcon.jsx (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/Components/BidTracker/BidStep/BidStepIcon/BidStepIcon.jsx:59]: Indeed!

File: src/Constants/BidData.js (hunk 3 above)
Comment (elizabeth-jimenez) [reply src/Constants/BidData.js:79]: So "What is a "Drafts""?

File: src/Constants/BidData.js (hunk 4 above)
Comment (elizabeth-jimenez) [reply src/Constants/BidData.js:99]: Noice!

**Comment by elizabeth-jimenez** [comment]
- [ ] 2560*1440
- [ ] 1920*1080
- [ ] 1536*864
- [ ] 1440*900
- [ ] 1366*768
- [ ] 1280*720
- [ ] 1024*768

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 all done 🦜

File: src/Constants/BidData.js [hunk 5]
```diff
@@ -76,7 +76,7 @@ export const BID_TRACKER_SUBMITTED_ACTIVE_STATUSES = [
 export const BID_TRACKER_TOOLTIP_TEXT = {
   [DRAFT_PROP]: {
     title: `What is a "${GET_DRAFT_TITLE()}"?`,
-    text: `"${GET_DRAFT_TITLE()}" are bids you have on your bid tracker but you haven’t submitted them for consideration.`,
+    text: `"${GET_DRAFT_TITLE()}"s are bids you have on your bid tracker but you haven’t submitted them for consideration.`,
```
Comment (mjoyce91) [inline src/Constants/BidData.js:79]: @elizabeth-jimenez - the "s" should be inside the quotes

Comment (elizabeth-jimenez) [reply src/Constants/BidData.js:79]: 😂  Looking back at it, all the other tooltips are singular, so im thinking maybe this one should be too? Even the title is singular

Comment (elizabeth-jimenez) [reply src/Constants/BidData.js:79]: A “Draft” is a bid you have on your bid tracker but you haven’t submitted for consideration.
??

Comment (mjoyce91) [reply src/Constants/BidData.js:79]: Draft is a noun though, whereas the other are statuses

Comment (elizabeth-jimenez) [reply src/Constants/BidData.js]: 🤪  updated; good catch!

**Comment by SidelineCory24** [comment]
Looks good. All of the wording for `BidData.js` matches Jira as well. Nice work! I'll let Mike do the final approval.

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**